test/dlc_tests/container_tests/bin/pytorch_tests/testTorchdata

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-set -e
-
```

Comment:
Why merge this in with `set -e` removed?

Reply:
The test author's original intention was for the script to execute from start to end since there is logging code after the test run.